smoketest/tests/send_token_to_penpal.rs

```diff
@@ -63,6 +63,7 @@ async fn send_token_to_penpal() {
 
 	// Lock tokens into vault
 	let amount: u128 = U256::from(value).low_u128();
+	let fee: u128 = 30_000_000_000_000_000;
```

Comment:
There is an API `IGateway.quoteSendTokenFee` to retrieve the fee. I think I prefer to use that because then the test would verify that it returns a fee large enough.

Reply:
addressed in https://github.com/Snowfork/snowbridge/pull/1064/commits/dd7cb06727d41448bfaca30f1c5a1720127788c3